server/src/yml-objects/YmlEnumMember.ts

```diff
@@ -9,6 +9,11 @@ export class YmlEnumMember extends AbstractYmlObject {
 
     constructor(indentifier: string, uri: string) {
         super(indentifier, CompletionItemKind.EnumMember, uri);
+        this.kindName = 'enum member';
+    }
+
+    public setDetail(type: string) {
+        return `(${this.kindName}) ${this.getShortName} ⇒ ${type}`;
```

Comment:
Weird that you follow more or less the same pattern for every kind of symbol (“(foo) blahblah ⇒ plop”) but have to hardcode it in every class.